app/domain/person/filter/list.rb

```diff
@@ -44,7 +44,14 @@ def filter_with_selection
   def filter
     # When not filtering, the default is to exclude all passive and external people,
     # i.e. include only members
-    chain.present? ? chain.filter(list_range) : list_range.members
+
+    if chain.present?
+      chain.filter(list_range)
+    else
+      list_range.where(roles: { archived_at: nil })
+                .or(list_range.where(Role.arel_table[:archived_at].gt(Time.now.utc)))
```

Comment:
Unterstützen wir in der Zukunft liegende Archivierungsdaten wirklich?

Reply:
Hmm für mich macht es einfach von der Definition so Sinn.
Keine archivierten Gruppen -> archived_at: nil || in der Zukunft.
Unterstützen wir es aktuell: Nein. Zukünftig: Wahrscheinlich auch nicht. Ich finds einfach irgendwie "runder" :woozy_face: